suitcase/utils/conftest.py

```diff
@@ -0,0 +1,111 @@
+import bluesky # noqa
+from bluesky.tests.conftest import RE # noqa
+from bluesky.plans import count # noqa
+import event_model # noqa
+from ophyd.tests.conftest import hw # noqa
+import pytest # noqa
+from . import UnknownEventType # noqa
+
+hw = hw()
```

Comment:
This is itself a fixture and should be passed into other fixtures/tests so that you get hardware with fresh state every time.

Reply:
thanks resolved this